src/AddThreepid.js

```diff
@@ -162,6 +164,7 @@ export default class AddThreepid {
         try {
             if (await MatrixClientPeg.get().doesServerSupportSeparateAddAndBind()) {
                 if (this.bind) {
+                    const identityServerDomain = MatrixClientPeg.get().idBaseUrl.split("://")[1];
```

Comment:
I think this is left over from merging develop? This shouldn't be needed now, as `getIdServerDomain()` a few lines below takes care of it.

Reply:
ah yep, just noticed it was failing lint on these as you were reviewing